web/src/components/SearchBar.jsx

```diff
@@ -17,9 +17,9 @@ export default function SearchBar({ setSearchInput, searchInput }) {
           root: classes.inputRoot,
           input: classes.inputInput,
          }}
-         value={searchInput}
+         value={ searchInput}
```

Comment:
nit: If you put a leading space, then lets put a trailing one as well: `{ searchInput }`

Reply:
Thanks @maniramezan, trailing space has been added.